.circleci/config.yml

```diff
@@ -6,7 +6,10 @@ orbs:
 executors:
   linux-node:
     docker:
-      - image: circleci/node:stretch
+      - image: cimg/node:19.0.0
```

Comment:
Do we need both the docker image for node 19 and also to download it manually below?

Reply:
This is actually a separate change, I think i'll just revert it.  we don't run these new tests in the `linux-node` image.